packs/data/feats.db/six-pillars-stance.json

```diff
@@ -25,7 +25,7 @@
         },
         "rules": [],
         "source": {
-            "value": "Pathfinder #167: Ready? Fight!"
```

Comment:
If there are changes in a newer release, it should generally take precedence.
The updated source should be: "Pathfinder Fists of the Ruby Phoenix Hardcover Compilation"